inst/cpp_generation/model_input.csv

```diff
@@ -1,38 +1,38 @@
-artmx_timerr,mortaility_time_rate_ratio,real_type,,,FALSE,2,base.hTS,proj_years,,
-art_dropout,dropout,real_type,,,FALSE,1,proj_years,,,
-art15plus_num,adults_on_art,real_type,,,FALSE,2,base.NS,proj_years,,
-art15plus_isperc,adults_on_art_is_percent,int,,,FALSE,2,base.NS,proj_years,,
-,h_art_stage_dur,real_type,,0.5,FALSE,1,base.hTS - 1,,,
-scale_cd4_mort,scale_cd4_mortality,int,,,FALSE,1,1,,,
-art_alloc_mxweight,initiation_mortality_weight,real_type,,,FALSE,1,1,,,
-paed_incid_input,hc_nosocomial,real_type,ModelVariant::run_child_model,,FALSE,1,proj_years,,,
-paed_cd4_dist,hc1_cd4_dist,real_type,ModelVariant::run_child_model,,FALSE,1,children.hc2DS,,,
-paed_cd4_transition,hc_cd4_transition,real_type,ModelVariant::run_child_model,,FALSE,2,children.hc2DS,children.hc1DS,,
-paed_cd4_mort,hc1_cd4_mort,real_type,ModelVariant::run_child_model,,FALSE,3,children.hc1DS,children.hcTT,children.hc1AG,
-adol_cd4_mort,hc2_cd4_mort,real_type,ModelVariant::run_child_model,,FALSE,3,children.hc2DS,children.hcTT,children.hc2AG,
-paed_cd4_prog,hc1_cd4_prog,real_type,ModelVariant::run_child_model,,FALSE,1,children.hc1DS,,,
-adol_cd4_prog,hc2_cd4_prog,real_type,ModelVariant::run_child_model,,FALSE,1,children.hc2DS,,,
-ctx_effect,ctx_effect,real_type,ModelVariant::run_child_model,,FALSE,1,1,,,
-ctx_val,ctx_val,real_type,ModelVariant::run_child_model,,FALSE,1,proj_years,,,
-paed_art_elig_age,hc_art_elig_age,real_type,ModelVariant::run_child_model,,FALSE,1,proj_years,,,
-paed_art_elig_cd4,hc_art_elig_cd4,real_type,ModelVariant::run_child_model,,TRUE,2,options.p_idx_hiv_first_adult,proj_years,,
-mort_art_rr,hc_art_mort_rr,real_type,ModelVariant::run_child_model,,FALSE,3,base.hTS,options.p_idx_hiv_first_adult,proj_years,
-paed_art_mort,hc1_art_mort,real_type,ModelVariant::run_child_model,,FALSE,3,children.hc1DS,base.hTS,children.hc1AG,
-adol_art_mort,hc2_art_mort,real_type,ModelVariant::run_child_model,,FALSE,3,children.hc2DS,base.hTS,children.hc2AG,
-artpaeds_isperc,hc_art_isperc,int,ModelVariant::run_child_model,,FALSE,1,proj_years,,,
-paed_art_val,hc_art_val,real_type,ModelVariant::run_child_model,,FALSE,1,proj_years,,,
-init_art_dist,hc_art_init_dist,real_type,ModelVariant::run_child_model,,FALSE,2,options.p_idx_hiv_first_adult,proj_years,,
+basepop,Demography.base_pop,real_type,,,FALSE,2,base.pAG,base.NS,,
+Sx,Demography.survival_probability,real_type,,,FALSE,3,base.pAG + 1,base.NS,proj_years,
+netmigr_adj,Demography.net_migration,real_type,,,FALSE,3,base.pAG,base.NS,proj_years,
+asfr,Demography.age_specific_fertility_rate,real_type,,,FALSE,2,options.p_fertility_age_groups,proj_years,,
+births_sex_prop,Demography.births_sex_prop,real_type,,,FALSE,2,base.NS,proj_years,,
+incidinput,Incidence.total_rate,real_type,,,FALSE,1,proj_years,,,
+incrr_age,Incidence.relative_risk_age,real_type,,,FALSE,3,base.pAG - options.p_idx_hiv_first_adult,base.NS,proj_years,
+incrr_sex,Incidence.relative_risk_sex,real_type,,,FALSE,1,proj_years,,,
```

Comment:
@mwalte10 just FYI updated the name of this variable as I think I missed it off when doing big renaming. I think this now matches what we settled on in the README.

Note also the change in format here, now need to specify what struct this data belongs to